content/en/apps/reference/api.md

```diff
@@ -708,6 +708,395 @@ Content-Type: application/json
 }
 ```
 
+### GET /api/v1/people
+
+*Added in 4.11.0*
+
+Returns a JSON array of people based on the specified page parameters.
+
+#### Permissions
+
+`can_view_contacts`
+
+#### Query Parameters
+
+| Name        | Required | Description                                                                                                                                                                                                              |
+|-------------|----------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| person_type | true     | ID of the contact_type for the new person.                                                                                                                                                                               |
+| cursor      | false    | The token identifying which page to retrieve. A `null` value indicates the first page should be returned. Subsequent pages can be retrieved by providing the cursor returned with the previous page. Defaults to `null`. |
+| limit       | false    | The total number of people to fetch. Defaults to `100`.                                                                                                                                                                  |
+
+#### Examples
+
+1. Get people of type `person`.
+
+```
+GET /api/v1/person?personType=person
+```
+
+```
```

Comment:
When these code blocks have a specified language like `json` or something appropriate, then color highlighting and `copy to clipboard` functionality is also provided by hugo. Still, for consistency within the page, I refrained from doing this.

Reply:
![image](https://github.com/user-attachments/assets/4ceeebc0-5eb5-43db-a624-8d5ec4e959db)

Reply:
:+1: Ohh, that is nice!  But yeah, lets stay consistent for now.  Ultimately we want to replace this whole page with auto-generated Swagger docs anyway...